src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Insights/Add-ArmApplicationInsightsMetricAlertCriteriaODataType.ps1

```diff
@@ -0,0 +1,19 @@
+function Add-ArmApplicationInsightsMetricAlertCriteriaODataType {
```

Comment:
Is this method really necessary, could we add the logic into ArmApplicationInsightsMetricAlert.

Can you create a metric alert without calling the OData?